playbooks/charon-devel.yml

```diff
@@ -0,0 +1,30 @@
+- hosts: devel
+  become: true
+  vars:
+    katello_repositories_version: nightly
+    foreman_installer_scenario: katello-devel
+    foreman_installer_options_internal_use_only:
+      - "--disable-system-checks"
+      - "--katello-devel-github-username {{ katello_devel_github_username }}"
+      - "--katello-devel-enable-ostree=true"
+    foreman_installer_additional_packages:
+      - foreman-installer-katello-devel
+    foreman_installer_options:
+      - "--katello-devel-extra-plugins theforeman/foreman_remote_execution"
+      - "--katello-devel-extra-plugins theforeman/foreman_discovery"
+      - "--katello-devel-extra-plugins theforeman/foreman_ansible"
+      - "--katello-devel-extra-plugins theforeman/foreman-tasks"
```

Comment:
i think there is the same problem as with "foreman-docker".
using the current version, foreman installer is failing with:

```
[!] There was an error parsing `Gemfile`: Undefined local variable or method `tasks_dev' for Gemfile. Bundler cannot continue.
```
foreman-tasks.local.rb:
```
:development_group => :foreman-tasks_dev
```
enclosing the symbol in doublequotes works fine.

Reply:
https://github.com/theforeman/puppet-katello_devel/pull/145 fixes that but it has to propagate to the nightly repositories which is currently blocked by an issue being fixed in Foreman.